features/product/deleting_product.feature

```diff
@@ -0,0 +1,26 @@
+@product
+Feature: Deleting a product
+    In order to remove test or incorrect products
```

Comment:
We agreed on one version:
`In order to remove test, obsolete or incorrect {resource}`